interface/.dockerignore

```diff
@@ -0,0 +1,4 @@
+bin
+build
+node_modules
```

Comment:
`node_modules` is a cache (enormous dumping ground, really) where `npm` downloads its packages. The vast majority of what's in there is used for the toolchain that builds the interface. A small number of packages are plucked out and bundled into our minified JS payload as part of the final steps in the build.

Since it is downloaded for the build, nothing from the `node_modules` directory should appear in our repo. In fact, you'll see that our `.gitignore` lists `interface/node_modules/` to be sure we don't accidentally add any of it.